### PR TITLE
adding more correctness benchmarks

### PR DESCRIPTION
**Pull request by jycor**

### CODE REVIEW

**Review by zachmu — COMMENTED** [review]
LGTM

File: content/reference/sql/benchmarks/correctness.md [hunk 1]
```diff
@@ -68,3 +69,22 @@ did not run could not complete due to a timeout earlier in the run.
...
+|       263 |   438 |        60.045662 |
+
+## Skipped Engine Tests
+Here are the total number of tests skipped by the engine for 
+version `1.34.3`. These are tests that we know are failing for one
+reason or another, but haven't been able to fix yet.
+
+| Skipped | Total | Percent Skipped |
+|---------|-------|-----------------|
+|     248 | 42624 |        0.581831 |
```
Comment (timsehn) [inline content/reference/sql/benchmarks/correctness.md]: I think we do correctness here, like 99.42, with that precision

File: content/reference/sql/benchmarks/correctness.md [hunk 2]
```diff
@@ -68,3 +69,22 @@ did not run could not complete due to a timeout earlier in the run.
...
+
+## Function Coverage
+We also measure the coverage of the functions in the SQL engine. This
+is a measure of how many of the supported MySQL functions are also
+supported by Dolt. 
+
+Here are Dolt's function coverage results for version `1.34.3`.
+| Supported | Total | Percent Coverage |
+|-----------|-------|------------------|
+|       263 |   438 |        60.045662 |
```
Comment (timsehn) [inline content/reference/sql/benchmarks/correctness.md]: Round this one to the closest % like 60%

**Review by timsehn — APPROVED** [review]

File: content/reference/sql/benchmarks/correctness.md [hunk 3]
```diff
@@ -68,3 +69,22 @@ did not run could not complete due to a timeout earlier in the run.
...
+
+## Skipped Engine Tests
+Here are the total number of tests skipped by the engine for 
+version `1.34.3`. These are tests that we know are failing for one
+reason or another, but haven't been able to fix yet.
+
```
Comment (timsehn) [inline content/reference/sql/benchmarks/correctness.md:86]: I think a paragraph here about why these are harder to fix then sqllogictests:

1. Constructed by us as we built Dolt. Human identified edge cases.
2. Not watered down by passing tests that test the same thing
3. Augment areas not caught by sqllogictest

**Review by timsehn — COMMENTED** [review]
Additional comment
